FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -83,6 +88,7 @@ public class FhirToDataLakeOrchestratorJob : IJob
 
         public async Task<string> ExecuteAsync(IProgress<string> progress, CancellationToken cancellationToken)
         {
+            _diagnosticLogger.LogInformation($"Start executing FhirToDataLake orchestrator job {_jobInfo.GroupId}");
```

Comment:
use `_jobInfo.Id` instead of `_jobInfo.GroupId`, and same for others.

Reply:
Job Id should be used to distinguish different jobs ?